src/app/routes/api/external/v1/admin/forms/admin-forms.external.routes.ts

```diff
@@ -0,0 +1,26 @@
+import { Router } from 'express'
+
+import { rateLimitConfig } from '../../../../../../config/config'
+import { authenticateApiKey } from '../../../../../../modules/auth/auth.middlewares'
+import * as AdminFormController from '../../../../../../modules/form/admin-form/admin-form.controller'
+import { limitRate } from '../../../../../../utils/limit-rate'
+
+export const AdminFormsExternalRouter = Router()
+
+// All routes in this handler should be protected by authentication.
+AdminFormsExternalRouter.use(authenticateApiKey)
+
+AdminFormsExternalRouter.route('/')
+  /**
+   * List the forms managed by the user
+   * @security session
+   *
+   * @returns 200 with a list of forms managed by the user
+   * @returns 401 when user is not authorised
+   * @returns 422 when user of given id cannnot be found in the database
+   * @returns 500 when database errors occur
+   */
+  .get(
+    limitRate({ max: rateLimitConfig.externalApi }),
```

Comment:
This protects the endpoint overall, but we will eventually need a rate limit per api key, so don't have a situation where one bad actors blocks the API for everyone.

In fact, in the TODOs related for that project, I would love finally getting a centralized rate limit system.

Reply:
FRM-948